src/wazuh_db/schema_agents.sql

```diff
@@ -340,9 +340,19 @@ CREATE TABLE IF NOT EXISTS sync_info (
     n_completions INTEGER DEFAULT 0
 );
 
+CREATE TABLE IF NOT EXISTS vuln_cves (
+    name TEXT,
+    version TEXT,
+    architecture TEXT,
+    cve TEXT,
+    PRIMARY KEY (name, version, architecture, cve)
+);
+CREATE INDEX IF NOT EXISTS packages_id ON vuln_cves (name);
+CREATE INDEX IF NOT EXISTS cves_id ON vuln_cves (cve);
+
 BEGIN;
 
-INSERT INTO metadata (key, value) VALUES ('db_version', '7');
+INSERT INTO metadata (key, value) VALUES ('db_version', '8');
```

Comment:
Version 4.2 already upgraded to DB version 7. We should stay at 7.

Reply:
Fixed